### PR TITLE
[RW-4313][risk=no] Teardown cluster resources bucket

### PR DESCRIPTION
**Pull request by calbach**
This is cleanup since we've switched to instead serving static files. This simplifies our deployment process, and as a side-effect, bypasses a problem in preprod (preprod buckets are configured differently than other environments).

### CODE REVIEW

File: api/snippets-menu/README.md [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+
+As code snippets are updated in the source repository, we will want to
+periocially update the menus in the Workbench. For now, this process is fairly
+manual and can be improved going forwards (see RW-2665):
+
+Prerequisite: Must have `jq` installed (for pretty printing).
+
+1. In a separate directory, clone the snippets repo and build the menu config:
+    https://github.com/all-of-us/workbench-snippets/blob/master/CONTRIBUTING.md#auto-generation-of-jupyter-snippets-menu-configuration
+1. Run the following from the `workbench/api/cluster-resources` directory to pull in the updated JSON
```
Comment (jmthibault79) [inline api/snippets-menu/README.md:21]: update path

File: api/snippets-menu/README.md [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+https://github.com/all-of-us/workbench-snippets. AoU configures Leo clusters as
+follows to enable this:
+
+1. Enable the snippets_menu/main extension in ./initialize_notebook_cluster.sh
+1. Deploy a Jupyter UI extension to configure the menu with AoU-specific snippets
+
+## Updating Snippets
+
+As code snippets are updated in the source repository, we will want to
+periocially update the menus in the Workbench. For now, this process is fairly
```
Comment (jmthibault79) [inline api/snippets-menu/README.md:14]: might as well fix this typo

File: api/snippets-menu/README.md [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+manual and can be improved going forwards (see RW-2665):
+
+Prerequisite: Must have `jq` installed (for pretty printing).
+
+1. In a separate directory, clone the snippets repo and build the menu config:
+    https://github.com/all-of-us/workbench-snippets/blob/master/CONTRIBUTING.md#auto-generation-of-jupyter-snippets-menu-configuration
+1. Run the following from the `workbench/api/cluster-resources` directory to pull in the updated JSON
+
+    ```
+    ./import_json_from_snippets_repo.sh <path to workbench-snippets repo>
```
Comment (jmthibault79) [inline api/snippets-menu/README.md:24]: This script needs to be updated to copy to the correct destination.  output: 
```
Copied pretty printed ../../../aou-snippets/build/r_dataset_snippets_menu_config.json into /Users/thibault/src/aou/api/cluster-resources/r-dataset-snippets-menu.json
```

File: api/snippets-menu/README.md [hunk 4]
```diff
@@ -0,0 +1,52 @@
+# Snippets Menu
+
+AoU Clusters enable the [Snippets Menu extension](https://jupyter-contrib-nbextensions.readthedocs.io/en/latest/nbextensions/snippets_menu/readme.html)
+with custom AoU-specific code snippets. The snippets live in
+https://github.com/all-of-us/workbench-snippets. AoU configures Leo clusters as
+follows to enable this:
+
+1. Enable the snippets_menu/main extension in ./initialize_notebook_cluster.sh
```
[the remark this answers is not in the document]
Comment (calbach) [reply api/snippets-menu/README.md:8]: Yes, thanks - updated.

File: api/snippets-menu/README.md (hunk 2 above)
Comment (calbach) [reply api/snippets-menu/README.md:14]: Done

File: api/snippets-menu/README.md (hunk 1 above)
Comment (calbach) [reply api/snippets-menu/README.md:21]: Done

File: api/snippets-menu/README.md (hunk 3 above)
Comment (calbach) [reply api/snippets-menu/README.md:24]: Oops, thanks. I've now run this myself to validate.

File: api/snippets-menu/README.md [hunk 5]
```diff
@@ -0,0 +1,52 @@
...
+## Snippets Extension Implementation
+
+The snippets menu extension Javascript is generated dynamically at deploy time
+by `./build.rb` into the GAE static resources folder. This process takes as
+inputs the templated extension `snippets-menu.js.template` and the per-language
+collocated menu JSON files.
+
+## Local testing
+
+See instructions for local extension testing above.
```
[the remark this answers is not in the document]
Comment (calbach) [reply api/snippets-menu/README.md:39]: Done

File: api/snippets-menu/build.rb [hunk 6]
```diff
@@ -2,6 +2,7 @@
 
 require 'fileutils'
 require_relative "../../aou-utils/workbench"
+require_relative "../../../aou-utils/utils/common"
```
[the remark this answers is not in the document]
Comment (calbach) [reply api/snippets-menu/build.rb:5]: Oops, fixed - ran a demo server deploy as well to verify

**Comment by calbach** [comment]
Sorry - thought I'd already pushed.. Now actually addressed.

**Pull request merged**